generator/uscore/uscore_generator.rb

```diff
@@ -117,7 +118,8 @@ def create_authorization_test(sequence)
         authorization_test = {
           tests_that: "Server rejects #{sequence[:resource]} search without authorization",
           index: sequence[:tests].length + 1,
-          link: 'https://build.fhir.org/ig/HL7/US-Core-R4/CapabilityStatement-us-core-server.html#behavior'
+          link: 'https://build.fhir.org/ig/HL7/US-Core-R4/CapabilityStatement-us-core-server.html#behavior',
+          description: 'A server SHALL reject any unauthorized requests by returning an HTTP 401 unauthorized response code.'
```

Comment:
fixed

Reply:
I think this should stay as it was since that's what we're actually testing for. It's just odd that US Core has seemingly contradictory information so close together.

Reply:
Changed back to 401